package.json

```diff
@@ -13,5 +13,9 @@
   },
   "devDependencies": {
     "prettier": "2.2.1"
+  },
+  "dependencies": {
+    "@types/express": "^4.17.11",
+    "@types/passport": "^1.0.6"
   }
```

Comment:
Can you move these two types into devDependencies?

- Uninstall with `yarn remove`
- Re-install with `yarn add -D`